transports/azure-service-bus/configuration/full.md

```diff
@@ -61,7 +61,7 @@ The following settings are available to define how queues should be created:
  * `RequiresDuplicateDetection(bool)`: Specifies whether the queue should perform native broker duplicate detection, defaults to `false`.
  * `DuplicateDetectionHistoryTimeWindow(TimeSpan)`: The time period in which native broker duplicate detection should occur.
  * `SupportOrdering(bool)`: Best effort message ordering on the queue, defaults to `false`.
- * `DescriptionFactory(Func<string, ReadOnlySettings, QueueDescription>)`: A factory method that allows to create a `QueueDescription` object from the Azure Service Bus SDK. Use this factory method to override any (future) setting that is not supported by the Queues API.
+ * `DescriptionCustomizer(Action<QueueDescription>)`: A factory method that allows to modify a `QueueDescription` object created by the Azure Service Bus SDK. Use this factory method to override any (future) setting that is not supported by the Queues API.
```

Comment:
>  to override any (future) setting that is not supported by the Queues API.

 to override any (future) setting**s** that ~~is~~ **are** not supported by the Queues API.